### PR TITLE
Install bluetooth proxies and media players from firmware.esphome.io

### PR DESCRIPTION
**Pull request by balloob**
## Description:
Point installation at firmware.esphome.io for bluetooth proxies and media players

Requires

- [x] bluetooth proxies https://github.com/esphome/firmware/pull/49
- [x] media players https://github.com/esphome/firmware/pull/52
- [x] build https://github.com/esphome/firmware/actions/runs/6413865996

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: projects/installer.html [hunk 1]
```diff
@@ -398,7 +398,7 @@ <h3>For advanced users</h3>
 
 <div
   class="type type-media hidden"
-  data-manifest-root="https://esphome.github.io/media-players"
+  data-manifest-root="https://firmware.esphome.io/media-players"
```
Comment (jesserockz) [inline projects/installer.html]:
```suggestion
  data-manifest-root="https://firmware.esphome.io/media-player"
```

File: projects/installer.html [hunk 2]
```diff
@@ -679,13 +680,16 @@ <h3>For advanced users</h3>
...
+
+  // Handle selecting a device
   document.querySelectorAll("input.device-option").forEach((radio) =>
     radio.addEventListener("change", () => {
       const typeRoot = radio.closest(".type");
       const manifestRoot = typeRoot.dataset.manifestRoot;
       if (manifestRoot) {
         const button = typeRoot.querySelector("esp-web-install-button");
-        button.manifest = `${manifestRoot}/${radio.value}-manifest.json`;
+        // TODO this has to be adjusted for firmware repo.
```
Comment (jesserockz) [inline projects/installer.html]: TODO still needed?

Comment (balloob) [reply projects/installer.html]: Nope. 
```suggestion
```

**Review by jesserockz — APPROVED** [review]

**Comment by yousaf465** [comment]
currently, this message is displayed "Testing publishing ESPHome firmwares to Github Pages" on https://firmware.esphome.io/

**Comment by balloob** [comment]
We source the firmware from that location. The actual firmware installation page is in the ESPHome docs: https://esphome.io/projects/

**Pull request merged**